src/components/interaction-designer/block-types/Core_SetContactPropertyBlock.vue

```diff
@@ -1,80 +1,54 @@
 <template>
   <div class="core-set-contact-property-block">
-    <h3 class="block-editor-header">
-      {{ `flow-builder.${block.type}` | trans }}
-    </h3>
-
-    <fieldset :disabled="!isEditable">
-      <block-label-editor
-        :block="block"
-        @gearClicked="showSemanticLabel = !showSemanticLabel" />
-      <block-semantic-label-editor
-        v-if="showSemanticLabel"
-        :block="block" />
-      <block-name-editor :block="block" />
-
-      <slot name="extras" />
-
-      <contact-property-editor :block="block" />
-
-      <hr>
-      <block-output-branching-config
+    <base-block
         :block="block"
-        :has-exit-per-choice="false"
-        @branchingTypeChangedToUnified="handleBranchingTypeChangedToUnified({block})" />
-
-      <categorization :block="block" />
-
-      <hr>
-
-      <first-block-editor-button
         :flow="flow"
-        :block-id="block.uuid" />
-    </fieldset>
-
-    <block-id :block="block" />
+        @handleBranchingTypeChangedToUnified="handleBranchingTypeChangedToUnified({block})"
+        :showSemanticLabel="false"
+        :usesDefaultContactPropsEditor="usesDefaultContactPropsEditor">
+      <template slot="extras">
+        <contact-property-editor v-if="!$slots['extras']" :block="block" />
+        <slot v-if="$slots['extras']" name="extras" />
+      </template>
+      <template slot="branching">
+        <slot name="branching" v-if="usesDefaultBranchingEditor">
```

Comment:
These slots allow wrapping the component and overriding very small parts rather than extending and overriding the whole template (maybe using BaseBlock). In adding this I'm trying to avoid the issues we had copying changes from the community builder to voto5 when things like the NewFlow page changed (and things getting out of sync) perhaps this is overkill though.